plugins/techdocs/src/reader/components/TechDocsReaderPage/TechDocsReaderPage.test.tsx

```diff
@@ -66,6 +71,16 @@ const techdocsStorageApiMock: jest.Mocked<typeof techdocsStorageApiRef.T> = {
   syncEntityDocs: jest.fn(),
 };
 
+const PageMock = () => {
+  const { namespace, kind, name } = useParams();
+  return <>{`PageMock: ${namespace}#${kind}#${name}`}</>;
+};
+
+jest.mock('@backstage/core-components', () => ({
+  ...jest.requireActual('@backstage/core-components'),
+  Page: jest.fn(),
+}));
```

Comment:
CI fails with following error:

```
   ● <TechDocsReaderPage /> › should render a techdocs reader page with children

    mockConstructor(...): Nothing was returned from render. This usually means a return statement is missing. Or, to render nothing, return null.

      41 |   let value: RenderResult;
      42 |   await act(async () => {
    > 43 |     value = render(nodes, options);
         |                   ^
      44 |   });
      45 |   return value!;
      46 | }
```

We could permanently mock `<PageMock>` instead of having it on each test. Failing due to returning `undefined`